infrastructure/variables.tf

```diff
@@ -13,6 +13,12 @@ variable "component" {
   default     = "api"
   description = "Scheduling and Listing "
 }
+
+variable "deployment_namespace" {
```

Comment:
what is this for?

Reply:
As per discussion, used in shortEnv for preview envs